waiter/config-composite.edn

```diff
@@ -34,9 +34,20 @@
 
  ; ---------- Security ----------
 
- :authenticator-config {:kind :one-user
+ :authenticator-config {:kind :composite
+                        :composite {:factory-fn waiter.auth.composite/composite-authenticator
+                                    :authentication-schemes {:one-user {:authenticator-factory-fn waiter.auth.authentication/one-user-authenticator
+                                                                        ;; The user account used to launch services:
+                                                                        :run-as-user #config/env "WAITER_AUTH_RUN_AS_USER"}
+                                                             :saml {:authenticator-factory-fn waiter.auth.saml/saml-authenticator
+                                                                    :idp-cert-uri #config/env "SAML_IDP_CERT_URI"
+                                                                    :idp-uri #config/env "SAML_IDP_URI"}}
+                                    :default-scheme :one-user}
```

Comment:
`scheme` feels confusing, `auth-protocol` feels like a better match to me.

Reply:
done